core/src/main/java/org/elasticsearch/common/xcontent/json/JsonXContentGenerator.java

```diff
@@ -34,6 +36,7 @@
 
     protected final BaseJsonGenerator generator;
     private boolean writeLineFeedAtEnd;
+    private  static final DefaultPrettyPrinter.Indenter INDENTER = new DefaultIndenter("  ", "\n");
```

Comment:
extra space between private static

Also, can we use a reference to the concrete `DefaultIndenter` implementation here so that we can remove the `private static SerializedString LF` at the end of the class and use `INDENTER.getEol()` instead?